aws/requirements.txt

```diff
@@ -0,0 +1,9 @@
+boto3==1.7.58
+botocore==1.10.58
+docutils==0.14
+jmespath==0.9.3
+PySocks==1.5.7
+python-dateutil==2.7.3
+s3transfer==0.1.13
+six==1.11.0
+xsmtplib==1.0.0.dev1
```

Comment:
why do you need this?

Reply:
If you do pip freeze, it puts all the packets install even if those are dependencies. I just installed 'xsmtplib'

Reply:
ok